### PR TITLE
docs: add section for using docker

### PR DESCRIPTION
**Pull request by jon-hagg**
### Purpose
Add docker section, which is mainly a pointer to the plug repo which has more details. 

### Time to review
5 min

### CODE REVIEW

File: source/user/installation_guide.md [hunk 1]
```diff
@@ -44,14 +44,12 @@ In that case, you will [Fork][git guide] the [PowerSimData][powersimdata_github]
 Before using the scenario framework, you need to update the path to the variables in ***PowerSimData/powersimdata/utility/server_setup.py*** file. Note that the **LOCAL_DIR** does not need to be updated if having a **ScenarioData** directory in your home folder is a good place to store scenario data. You can now create, execute, extract and analyze scenarios. The PowerSimData [README][powersimdata_local] goes over these steps.
 
 
-## 3. Validate your Installation
-Make sure the [tox] package is installed and run from the root folder:
-```bash
-pip install tox
+## 3. Using docker
+This option enables running the full framework in containers, colocated on the same machine.
+In this case, the only system requirement is to have docker, (including docker-compose) installed.
```
Comment (rouille) [inline source/user/installation_guide.md:49]: should we add a link to docker here too?

Comment (jon-hagg) [reply source/user/installation_guide.md:49]: Guess it wouldn't hurt, I could link to this landing page? https://docs.docker.com/get-docker/

Comment (rouille) [reply source/user/installation_guide.md:49]: Looks good

**Review by rouille — APPROVED** [review]
Looks good

**Pull request merged**